### DISCUSSION TITLE
How to add prefix when `sudo make install`

### DISCUSSION THREAD

**Discussion by Firestar-Reimu** [iraf-community/iraf#389] [Q&A] [answered]
The installation using **make install** recognizes the following
environment variables:

 * **prefix** - Common installation prefix (default: `/usr/local`). The
   installation goes to `${prefix}/lib/iraf`, with symlinks for the
   user-callable executables in `${prefix}/bin`, and the manpages
   installed in `${prefix}/share/man`. The `iraf` variable is set to
   `${prefix}/lib/iraf/`.

I am going to install it to `/usr/`, but I don't know what can I run:

`make install prefix=/usr/`

or:

`make install --prefix=/usr/`

or:

`make install -C /usr/`

**Comment by olebole** [comment, marked as answer]
`prefix` is a recognized environment variable, so you can use
```shell
prefix=/usr make install
```

You can also set it separately (f.e. if other tools want to use it as well):
```shell
export prefix=/usr
make install
```

**Comment by Firestar-Reimu** [comment]
it seems that `prefix=/usr make install` `make prefix=/usr install` `make install prefix=/usr` are all OK?

**Reply by olebole** [reply]
They will all work, yes.
